cypress/support/nodeManager/commands.ts

```diff
@@ -0,0 +1,28 @@
+import type { OnboardedNode } from "../../plugins/nodeManager/shared";
+import { Commands } from "../../plugins/nodeManager/shared";
+
```

Comment:
The nesting of this file in the `nodeManager` directory seems kind of redundant. We could just name the file `cypress/support/NodeManager.ts`.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1492/commits/21b8021537085b85d2b39bb8e6c86bc19caeb581